docs/BreakingChanges/021 WorkFlow 3.0 types are obsolete.md

```diff
@@ -0,0 +1,22 @@
+## 21: WorkFlow 3.0 types are obsolete
```

Comment:
{
  "Id": "21",
  "Title": "WorkFlow 3.0 types are obsolete",
  "Details": "Windows Workflow Foundation (WWF) 3.0 APIs (those from the System.Workflow namespace) are now obsolete.",
  "Suggestion": "New WWF 4.0 APIs (in System.Activities) should be used instead. An example of using the new APIs can be found [here](https://msdn.microsoft.com/en-us/library/jj205427.aspx) and further guidance is available [here](http://blogs.msdn.com/b/workflowteam/archive/2012/02/08/deprecatingwf3.aspx). Alternatively, since the WWF 3.0 APIs are still supported, they may be used and the build-time warning avoided either by suppressing it or by using an older compiler.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887#wwf",
  "Markdown": null,
  "ApplicableApis": [
    "N:System.Workflow"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": true,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 1
}

**Parsed APIs:**
- [ ] N:System.Workflow